include/soci/oracle/soci-oracle.h

```diff
@@ -278,6 +279,77 @@ struct oracle_session_backend : details::session_backend
             " where table_name = :t";
     }
     
+    virtual std::string create_column(const std::string & columnName, data_type dt,
+        int precision, int scale)
+    {
+        //  Oracle-specific SQL syntax:
+        
+        std::string res = columnName + " ";
```

Comment:
Let the users add quotes on their own, they have impact on later queries.
Note also that this feature is not intended to enable random junk or something that come from user input as column names (this does not make sense in DDL anyway), so we can assume that column names will be chosen wisely and most likely by the application developers.

You other comments were fixed in the new commit.